src/main/java/core/basesyntax/db/Storage.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax.db;
+
+import java.util.HashMap;
+
+public class Storage {
+    private static HashMap<String,Integer> fruits;
+
+    private Storage() {
+        this.fruits = new HashMap<>();
+    }
+
+    public static Storage of() {
+        return new Storage();
+    }

```

Comment:
This is rather strange combination. It's not singleton but have private constructor for unknown reason.
On this stage just use simple no-args constructor